azure/storage/blob/baseblobservice.py

```diff
@@ -215,14 +215,18 @@ def __init__(self, account_name=None, account_key=None, sas_token=None,
         self.key_encryption_key = None
         self.key_resolver_function = None
 
-    def make_blob_url(self, container_name, blob_name, protocol=None, sas_token=None):
+    def make_blob_url(self, container_name, blob_name,
+                      snapshot=None, protocol=None, sas_token=None):
         '''
         Creates the url to access a blob.
 
         :param str container_name:
             Name of container.
         :param str blob_name:
             Name of blob.
+        :param str snapshot:
+            The snapshot parameter is an opaque DateTime value that
```

Comment:
@yanheven, would you please apply @rambo-msft 's suggestion here and submit a final version? 
For sake of consistency in documentation :)
"An opaque DateTime value that when present, returns the URL of the snapshot" 

I will merge the PR as soon as this final fix is in.